whole-tale/dashboard
# Create tale modal

This PR should cover the features described in issue #452

1. Implementing the "Create Tale" modal
1. Adding "Create New Tale" button. This can go on the Browse page for now until Browse is refactored.
1. Removing the "Compose" panel 
1. prepopulates the modal based on query string parameters 
    1. go to /browse?environment=RStudio+%28rocker%2Fgeospatial%29&name=Satellite+Telemetry+Dataset+%28Raw%29%3A+Juvenile+Bearded+and+Spotted+Seals%2C+2004-2006%2C+Kotzebue%2C+Alaska.&uri=10.24431%2Frw1k118
1. a dropdown button with a default action to create new tale and launch

It does not cover:
1. Does not track tale imports - see comments on issue #452
1. Does not implement the "Select data" button in the modal - it does nothing. That was a separate issue #453 and will be included in that PR and not this one.

To test: 
`git fetch origin`
`git checkout -b create_tale_modal origin/create_tale_modal`
Navigate to /browse page and click "Create New Tale".

For documentation on how to use this modal see:
[/app/components/create-tale-modal/README.md ](https://github.com/whole-tale/dashboard/blob/create_tale_modal/app/components/ui/create-tale-modal/README.md)

For an example on how to use this modal see:
[/app/templates/browse.hbs](https://github.com/whole-tale/dashboard/blob/create_tale_modal/app/templates/browse.hbs)
